docs/public/_redirects

```diff
@@ -1,2 +1,3 @@
 /ph/*        https://astro-houston-ph.pages.dev/ph/:splat  200
 /:lang/*    /:lang/404/   404
+/zh/*       /zh-cn/:splat  200
```

Comment:
How to test the redirect?

Reply:
Redirect not working, getting HTTP 404 on this page: https://deploy-preview-855--astro-starlight.netlify.app/zh/getting-started/

Reply:
I got it. It's order!